pywavefront/visualization.py

```diff
@@ -56,28 +56,28 @@ def same(v):
 }
 
 
-def draw(instance):
+def draw(instance, unlit=False, useTexture=True):
```

Comment:
`use_texture` is better here to make parameters consistent with the rest of the library.

Reply:
What about `lighting_enabled=True` and `textures_enabled=True`? It's a bit more verbose, but will better explain to the user what these parameters are. Right now the first parameter is a negative and the seconds is a positive. Can be confusing.